common/aws/common.go

```diff
@@ -25,22 +25,38 @@ import (
 )
 
 const (
-	// ResourceType is the Boskos Type for AWS accounts
-	ResourceType = "aws-account"
-
 	// UserDataAccessIDKey is the key in UserData containing the AWS Access key ID
 	UserDataAccessIDKey = "access-key-id"
 
 	// UserDataSecretAccessKey is the key in UserData containing the AWS Secret access key
 	UserDataSecretAccessKey = "secret-access-key"
 )
 
+func GetSupportedAWSBoskosResourceTypes() []string {
+	// Supported Resource Types for Boskos AWS accounts
```

Comment:
rather than hard-coding this list in here, maybe it'd be better to pass a comma-delimited list of resources types as a flag? that way we wouldn't need any additional changes if another aws-account resource type was added.